app/components/SignUp/messages.js

```diff
@@ -56,6 +64,14 @@ export default defineMessages({
     id: `${scope}.blankPhoneError`,
     defaultMessage: 'Phone number is required',
   },
+  minLengthPhonevalidator: {
```

Comment:
Please check camelCase names.

```suggestion
  minLengthPhoneError: {
```